src/styles/blocks/header.scss

```
@@ -19,6 +17,7 @@
     display: flex;
     margin: 0;
     padding: 0;
+    transition: 300ms, color;
```

```suggestion
    transition: color 300ms;
```
Don't hardcore the time. Use variable instead